target/linux/ramips/image/mt7620.mk

```diff
@@ -131,6 +131,7 @@ define Device/bocco
   DTS := BOCCO
   DEVICE_TITLE := YUKAI Engineering BOCCO
   DEVICE_PACKAGES := kmod-sound-core kmod-sound-mt7620 kmod-i2c-ralink
+  SUPPORTED_DEVICES += planex,cs-qr10
```

Comment:
You don't actually need this. The first compatible string is used as board name only if it's not specified in target/linux/ramips/base-files/lib/ramips.sh

Reply:
Ah... This problem exists in all your current changes....